index.html

```diff
@@ -2647,6 +2648,88 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="generic">
+			<rdef>generic</rdef>
+			<div class="role-description">
+				<p>A container <a>element</a> that has no special meaning on its own, but it represents its children, and it can be styled.</p>
+				<p>The <pref>aria-textseparation</pref> <a>attribute</a> of a <rref>generic</rref> indicates how its text content is separated from the text content of neighboring elements.</p>
```

Comment:
I changed the sentence about aria-textseparation to:
> The aria-textseparation attribute of a generic indicates how its text content is separated from adjacent text content of adjacent generic elements.

Note that this sentence has been temporarily removed from this PR, because aria-textseparation has been moved to PR https://github.com/w3c/aria/pull/996. The sentence was kept in a comment in the other PR.